medusa/clients/torrent/utorrent_client.py

```diff
@@ -135,7 +188,8 @@ def _set_torrent_priority(self, result):
 
     def _set_torrent_pause(self, result):
         return self._request(params={
-            'action': 'pause' if app.TORRENT_PAUSED else 'start',
+            #  "stop" torrent, can always be resumed!
+            'action': 'stop' if app.TORRENT_PAUSED else 'start',
```

Comment:
Should we change to stop? May cause issues with users expecting old behaviour.

Reply:
Yes, it should be changed. The end result would be the exact same (torrent added, but not downloading). It can be resumed  as well.  "pause" state is not recommended in uTorrent, has been removed from uTorrent's toolbar for good reasons (being harmful to the swarm).

Reply:
@p0psicles thoughts?

Reply:
I think he's right. don't see any harm here, stopping it. Although I don't know exactly what the difference is with a paused state.